static/logos.json

```diff
@@ -540,6 +540,9 @@
     "name": "Prototype"
   },
   {
+    "name": "Protractor"
+  },
+  {
```

Comment:
Thanks. There now seems to be extra file included `angular-protractor.png`. Can you remove that?